kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Linker.kt

```diff
@@ -57,12 +60,24 @@ internal class Linker(val context: Context) {
 
         val libraryProvidedLinkerFlags = context.llvm.allNativeDependencies.map { it.linkerOpts }.flatten()
 
+        val preLinkContext = NativePreLinkContext(
+            config = context.config,
+            backendContext = context,
+            objcExportNamer = context.objCExport.namer,
```

Comment:
This doesn't work. In the link execution, the `namer` is never initialized. Need more research.